samples-generator/lib/Mesh.js

```diff
@@ -130,6 +136,23 @@ Mesh.prototype.setPositionsRelativeToCenter = function() {
     }
 };
 
+/**
+ * Bake materials as vertex colors. Use the default white opaque material.
+ */
+Mesh.prototype.transferMaterialToVertexColors = function() {
+    var material = this.material;
+    this.material = whiteOpaqueMaterial;
+    var vertexCount = this.getVertexCount();
+    var vertexColors = new Array(vertexCount * 4);
+    this.vertexColors = vertexColors;
+    for (var i = 0; i < vertexCount; ++i) {
```

Comment:
What do you think about using different per-vertex colors to give it a "colorful" feel? I feel like that would make a better sample.

Reply:
That would demonstrate vertex colors better, but I think this is alright for now. It's also easier to work with for the color blend mode test in Cesium which requires the center building to be red.